### PR TITLE
Improve the extension docs by providing more information

### PR DESCRIPTION
**Pull request by gomathyk**
## Purpose
To make further changes suggested in the Siddhi-execution-map extension.

## Goals
To make modifications/ additions to siddhi-execution-map in an attempt to come up with the relevant information related to all the extensions.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? N/A
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/execution/map/CreateFunctionExtension.java [hunk 1]
```diff
@@ -61,11 +61,14 @@
...
+        returnAttributes = @ReturnAttribute(description = "Returns the created map object. ", type = DataType.OBJECT),
 
         examples = @Example(
-                description = "returns a map with keys 1, 2, 3 and corresponding values \"one\", \"two\", \"three\"",
-                syntax = "create(1 , ”one” ,  2 , ”two” , 3 , ”three”)"),
-        returnAttributes = @ReturnAttribute(description = "Returns the created map object. ", type = DataType.OBJECT)
+                syntax = "create(1 , ”one” ,  2 , ”two” , 3 , ”three”)",
+                description = "This returns a map with keys 1, 2, 3 mapped with their corresponding values," +
+                        " \"one\", \"two\", \"three\"")
+
```
Comment (niveathika) [inline component/src/main/java/org/wso2/extension/siddhi/execution/map/CreateFunctionExtension.java]: Please remove unnecessary new lines

File: component/src/main/java/org/wso2/extension/siddhi/execution/map/GetFunctionExtension.java [hunk 2]
```diff
@@ -42,26 +42,30 @@
...
-                type = DataType.OBJECT)
+                description = "This returns the value object from the map that corresponds to the given key.",
+                type = DataType.OBJECT),
+
+        examples = @Example(
+                syntax = "get(company,1)",
+                description = "This function returns the value that is associated with the key, i.e., 1, from a " +
+                        "map named company.")
 )
+
```
Comment (niveathika) [inline component/src/main/java/org/wso2/extension/siddhi/execution/map/GetFunctionExtension.java]: Same

**Review by niveathika — CHANGES_REQUESTED** [review]

File: component/src/main/java/org/wso2/extension/siddhi/execution/map/CreateFunctionExtension.java (hunk 1 above)
Comment (gomathyk) [reply component/src/main/java/org/wso2/extension/siddhi/execution/map/CreateFunctionExtension.java]: Done

File: component/src/main/java/org/wso2/extension/siddhi/execution/map/GetFunctionExtension.java (hunk 2 above)
Comment (gomathyk) [reply component/src/main/java/org/wso2/extension/siddhi/execution/map/GetFunctionExtension.java]: Done

**Review by niveathika — APPROVED** [review]

**Pull request merged**